config/fabric_module_set_admin_status.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2024, Nokia
+# All rights reserved.
+
+import re
+import subprocess
+import time
+from swsscommon.swsscommon import SonicV2Connector
+from sonic_py_common.logger import Logger
+from platform_ndk import nokia_common
```

Comment:
Should not have any vendor specific import here. sonic-utilities is sonic generic.